.github/workflows/deploy.yaml

```diff
@@ -2,14 +2,19 @@ name: deploy
 on:
   push:
     branches: [master]
+    tags: [v*]
 jobs:
   deploy:
     runs-on: ubuntu-latest
     env:
-      AUTODEPLOY_TAG: master
-      DOCKERHUB_PROJECT: etherbalance
+      AUTODEPLOY_URL: ${{ secrets.AUTODEPLOY_URL }}
       DOCKER_NAME: ${{ secrets.DOCKER_NAME }}
       DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      REGISTRY_URI: "gnosispm/etherbalance"
     steps:
-      - uses: actions/checkout@v2
-      - run: docker/deploy.sh $AUTODEPLOY_TAG
+      - name: Deploy Staging
+        if: github.ref == 'refs/heads/master'
+        run: bash docker/deploy.sh staging
+      - name: Deploy Tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: bash docker/deploy.sh ${GITHUB_REF##*/}
```

Comment:
What do these mean? Do we have two different types of deployments? I would think we only want to deploy when a new tag is made.

@giacomolicari